utils/calibration_utils.py

```diff
@@ -0,0 +1,32 @@
+from contextlib import contextmanager
+import time
+
+
+def set_calibration_file(channel_access, filename):
+    """
+    Sets a calibration file. Retries if it didn't set properly first time.
+    """
+    max_retries = 10
+
+    for _ in range(max_retries):
+        channel_access.set_pv_value("CAL:SEL", filename)
+        channel_access.assert_that_pv_alarm_is("CAL:SEL", channel_access.Alarms.NONE)
+        time.sleep(3)
```

Comment:
can we process the pv here instead of manually sleeping?

Reply:
Good suggestion but in this case I'm not 100% sure where the slow down is coming from so I think we'll err on the side of stability over speed.